hashira-web/src/firebase.ts

```diff
@@ -20,7 +20,7 @@ import { v4 as uuidv4 } from "uuid";
 
 const firebaseConfig = {
   apiKey: "AIzaSyDMkM3qb_CUokFQDSFemLhPOqXJrR-rVbo",
-  authDomain: "hashira-web.firebaseapp.com",
+  authDomain: "hashira-web.web.app",
```

Comment:
```suggestion
  authDomain: "web.app",
```


for stagings?

Reply:
It seems authDomain for staging is automatically added by misterious power
![image](https://github.com/pankona/hashira/assets/6533008/54b1358b-6168-4828-9f25-50e441c0875c)